docs/features/tenancy.md

```diff
@@ -0,0 +1,13 @@
+# Tenancy
+
+Multi-tenancy is implemented via assigning VMs and Datastore to a tenant. A tenant can be granted access to create, delete or mount volumes on a specific datastore. VMs assigned to a tenant can then execute Docker volume APIs on an assigned datastores. Within a datastore multiple tenants can store their Docker volumes. A tenant cannot access volumes created by a different tenant, i.e. tenants have their own independent namespace even if tenants share datastores. VMs cannot be shared between tenants.
```

Comment:
Ok will think about it. I do not want to explain tenancy itself but just how we implement it.